tests/features/generated_story_tables.feature

```diff
@@ -90,3 +96,4 @@ Scenario: Format of story table is v1
     | textarea | | Multiline text\narea value |
     | show_3 | True | true |
     | show_2 | True | true |
+    |  |  | /sign |
```

Comment:
I guess this should technically have two signatures. The table rows no longer get removed, though, so it's not technically necessary. If people were still going to use old tables, this may or may not be considered a problem. Have to work on the generator to see how it needs to be updated.